e2e/test/scripts/create_cluster.sh

```diff
@@ -7,7 +7,8 @@ set -o nounset
 export LINODE_API_TOKEN="$1"
 export CLUSTER_NAME="$2"
 export IMAGE="$3"
-
+export REGION="$4"
+export K8S_VERSION="$4"
```

Comment:
This one should be argument $5

Reply:
Oh good catch! As you can tell we don't really use this script so I did not notice.